schema-sandbox/schema.json

```diff
@@ -64,6 +126,8750 @@
+                },
+                "department": {
+                    "type": "string"
+                },
+                "doi": {
+                    "$ref": "#/definitions/doi"
+                },
+                "identifiers": {
+                    "items": {
+                        "$ref": "#/definitions/identifier"
+                    },
+                    "minItems": 1,
+                    "type": "array",
+                    "uniqueItems": true
+                },
+                "edition": {
+                    "type": "string"
+                },
+                "editors": {
+                    "items": {
+                        "anyOf": [
+                            {"$ref": "#/definitions/person"},
+                            {"$ref": "#/definitions/entity"}
+                        ]
+                    },
+                    "minItems": 1,
+                    "type": "array",
+                    "uniqueItems": true
+
+                },
+                "editors-series": {
+                    "items": {
+                        "anyOf": [
+                            {"$ref": "#/definitions/person"},
+                            {"$ref": "#/definitions/entity"}
+                        ]
+                    },
+                    "minItems": 1,
+                    "type": "array",
+                    "uniqueItems": true
+                },
+                "end": {
+                    "type": "integer"
+                },
+                "entry": {
+                    "type": "string"
+                },
+                "filename": {
+                    "type": "string"
+                },
+                "format": {
+                    "type": "string"
+                },
+                "institution": {
+                    "$ref": "#/definitions/entity"
+                },
+                "isbn": {
+                    "pattern": "^(?:ISBN(?:-1[03])?:? )?(?=[0-9X]{10}$|(?=(?:[0-9]+[- ]){3})[- 0-9X]{13}$|97[89][0-9]{10}$|(?=(?:[0-9]+[- ]){4})[- 0-9]{17}$)(?:97[89][- ]?)?[0-9]{1,5}[- ]?[0-9]+[- ]?[0-9]+[- ]?[0-9X]$",
+                    "type": "string"
+                },
+                "issn": {
+                    "pattern": "^\\d{4}-\\d{3}[\\dxX]$",
+                    "type": "string"
+                },
+                "issue": {
+                    "type": "string"
+                },
+                "issue-date": {
+                    "type": "string"
+                },
+                "issue-title": {
+                    "type": "string"
+                },
+                "journal": {
+                    "type": "string"
+                },
+                "keywords": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "uniqueItems": true,
+                    "minItems": 1
+                },
+                "languages": {
```

Comment:
Not sure the language enum is worth the (performance) trouble, I would be ok to remove the key or allow free form string of e.g maxLength 3. Note the three character set is already using ~50% of the total space occupied by 26^3 possibilities

Reply:
Agree, let's do `"maxLength": 3`.

Reply:
Done in https://github.com/citation-file-format/citation-file-format/commit/5f7a21a3428e4ecc81b5c3a3e81201144c2f169f.

Reply:
Greate. I've additionally created #127 so we can discuss that separately and it';s not bloicking merge of this PR